### PR TITLE
Deploy dev

### PR DESCRIPTION
**Pull request by netwmr01**
### Contributing guide

https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md
### Changelog

*
*
*
*
*

### CODE REVIEW

**Comment by netwmr01** [comment]
@mdzz

File: cloudera-on-centos/README.md [hunk 1]
```diff
@@ -3,12 +3,12 @@ Once the trial has concluded, the Cloudera Enterprise features will be disabled
 
 # By clicking "Deploy to Azure" you agree to the Terms and Conditions below.
 # DS14 Deployment(use this if you are not sure)
-<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fcloudera-on-centos%2Fazuredeploy.json" target="_blank">
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fnetwmr01%2Fazure-quickstart-templates%2Fdeploy-dev%2Fcloudera-on-centos%2Fazuredeploy.json" target="_blank">
```
Comment (mdzz) [inline cloudera-on-centos/README.md:6]: I assume this change will be reverted once we post a PR to the main MSFT repo later?

**Review by mdzz — APPROVED** [review]
Just have one small question. The rest of the changes look good.

File: cloudera-on-centos/README.md (hunk 1 above)
Comment (netwmr01) [reply cloudera-on-centos/README.md:6]: that is right.
This PR will be closed.
A new commit with reset of the change will go to master

**Pull request closed without merging**